### ISSUE TITLE
Show form and cookies variables in error details view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#37]
```
I'm using the xml logger. A lot of the time, the information in the details
view is enough to figure out if I need to fix something in the code. But
occasionally I need the form variables to figure out what really happened.
Now I have to open up the right xml file and look at the info. It would be
terrific if the details page would include that information if it existed
in the error entry.

```

Original issue reported on code.google.com by `donnie.h...@gmail.com` on 3 Oct 2007 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
This has already been addressed via issue #15. On the detailed page of each 
error, 
you'll now find an additional link that will render the underlying XML of the 
error. 
In there you can find all the details related to forms, cookies and what not. 
This 
was originally not included as a nicely formatted HTML view because the 
view-state 
(as well as some other large form fields) in some applications can be 
horrendously 
big and awkward to display, providing little value most of the time. For the 
rare 
moments that you need to dig into full context, however, the raw XML should 
suffice 
now. Hopefully you're okay with that.

BTW, cookies are visible on the error detail page right now via the HTTP_COOKIE 
server variable although in a very raw format.
```

Original comment by `azizatif` on 3 Oct 2007 at 2:29
- Changed title: **Show form and cookies variables in error details view**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I'm assuming the above is acceptable and closing this issue for now.
```

Original comment by `azizatif` on 3 Oct 2007 at 7:10
- Changed state: **WontFix**
